bukkit/src/main/java/com/github/games647/fastlogin/bukkit/hook/SodionAuthHook.java

```diff
@@ -68,6 +70,9 @@ public boolean forceRegister(Player player, String password) {
             plugin.getLog().warn("Currently SodionAuth is not accepting forceRegister, " +
                     "It may be caused by unsupported AuthBackend");
             return false;
+        } catch (InterruptedException | ExecutionException e) {
+            e.printStackTrace();
```

Comment:
Could you make use of the logger with a short message what we were trying here? At least we should give the users some hints about the error. Something like "Unexpected error during ..." should already do it.